stdlib/src/ssl.rs

```diff
@@ -539,12 +539,12 @@ mod _ssl {
 
         #[pygetset]
         fn options(&self) -> libc::c_ulong {
-            self.ctx.read().options().bits()
+            self.ctx.read().options().bits().try_into().unwrap()
```

Comment:
```suggestion
            self.ctx.read().options().bits() as _
```
maybe this will be fine